rules/policy.go

```diff
@@ -535,6 +550,8 @@ func appendProtocolMatch(match iptables.MatchCriteria, protocol *proto.Protocol,
 	return match
 }
 
+var SkipRule = errors.New("Rule skipped")
+
 func (r *DefaultRuleRenderer) CalculateRuleMatch(pRule *proto.Rule, ipVersion uint8) (iptables.MatchCriteria, error) {
```

Comment:
Looks like this function no longer returns an error in any situation so why not remove the error argument and the error check above?